tests/helpers/test_translation.py

```diff
@@ -187,11 +190,15 @@ def mock_load_translation_files(files):
         "homeassistant.helpers.translation.async_get_integration",
         return_value=integration,
     ):
-        translations = await translation.async_get_translations(hass, "en", "hello")
+        tasks = [
+            translation.async_get_translations(hass, "en", "hello") for _ in range(5)
+        ]
+        all_translations = await asyncio.gather(*tasks)
 
-    assert translations == {
+    assert all_translations[0] == {
         "component.component1.hello": "world",
     }
+    assert load_count == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)